contracts/src/v0.6/Flags.sol

```diff
@@ -18,6 +18,7 @@ import "./interfaces/FlagsInterface.sol";
 contract Flags is FlagsInterface, SimpleReadAccessController {
 
   AccessControllerInterface public raisingAccessController;
+  AccessControllerInterface public loweringAccessController;
 
   mapping(address => bool) private flags;
```

Comment:
If we are doing a major refactor, we could change `address` for `bytes32` for more flexibility